modules/git/bootstrap.Makefile

```diff
@@ -4,28 +4,28 @@ ifeq ($(wildcard .git),)
   endif
 else
 
-GIT ?= $(shell which git)
+GIT := $(shell which git)
```

Comment:
Is there no way to fix this for make 4.4 and retain the `?=` form?

Reply:
I can try doing something like:
```
PATHWISE_GIT := $(shell which git)
GIT ?= $(PATHWISE_GIT)
```

but is that realistically something we'd want to give folks configuration over? that is to say, I'd question when would folks need to override the path of git? And if they did so, would there be a need to consistently do so?